### PR TITLE
Run button

### PR DESCRIPTION
**Pull request by amanshawar**
- [x] make proxy
- [x] run the two servers
- [x] update trainerName
- [x] add to database !

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -15,7 +16,7 @@
     "styled-components": "^4.1.3"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "BROWSER=none react-scripts start",
```
Comment (m4v15) [inline client/package.json]: why?

File: src/controllers/activity.controller.js [hunk 2]
```diff
@@ -6,12 +6,13 @@ const add = (req, res) => {
   const {
     name, hours, startDate, endDate, trainerName, location, program, objective,
   } = req.body;
-
+console.log(name, hours, startDate, endDate, trainerName, location, program, objective);
```
Comment (m4v15) [inline src/controllers/activity.controller.js]: is this needed?

File: client/package.json (hunk 1 above)
Comment (m4v15) [reply client/package.json]: When you do new or unusual things, its a good idea to explain why you have done them in the description of the pull request

File: client/src/components/Form/index.js [hunk 3]
```diff
@@ -29,79 +33,93 @@ class MainForm extends Component {
     });
   };
 
+  handleChange= ({target:{name,value}})=> {
```
Comment (jema28) [inline client/src/components/Form/index.js]: Why is this handle change here? The point of Formik is to remove the need to write these yourselves.

File: client/src/components/Form/index.style.js [hunk 4]
```diff
@@ -3,6 +3,9 @@ import { Form, Field } from "formik";
 import "react-datepicker/dist/react-datepicker.css";
 import DatePicker from "react-datepicker";
 
+export const StyledContainerBtn=styled.div`
+margin-right:422px;
```
Comment (jema28) [inline client/src/components/Form/index.style.js]: This isn't the way to place items of the page! Look into using flex or absolute positioning.

File: src/controllers/activity.controller.js (hunk 2 above)
Comment (jema28) [inline src/controllers/activity.controller.js]: Please delete this console log, how did it get through your eslinting?

**Pull request merged**